packages/esm-service-queues-app/src/queue-rooms/queue-room-form.workspace.tsx

```diff
@@ -10,156 +10,175 @@ import {
-                      {location.name}
-                    </SelectItem>
-                  ))}
-              </Select>
-            </section>
+            <Controller
+              name="queueLocation"
+              control={control}
+              render={({ field: { onChange, value, ...field } }) => (
+                <Select
+                  {...field}
+                  value={value || ''}
+                  onChange={(e) => {
+                    onChange(e.target.value);
+                  }}
+                  labelText={t('queueLocation', 'Queue location')}
+                  id="location"
+                  invalidText={errors.queueLocation?.message}
+                  invalid={!!errors.queueLocation}>
+                  <SelectItem text={t('queueLocation', 'Queue location')} value="" />
+                  {queueLocations.length > 0 &&
+                    queueLocations?.map((location) => (
+                      <SelectItem key={location.id} text={location.name} value={location.id}>
+                        {location.name}
+                      </SelectItem>
+                    ))}
+                </Select>
+              )}
+            />
           </Layer>
         </Column>
 
         <Column>
           <Layer className={styles.input}>
-            <section className={styles.section}>
-              <Select
-                labelText={t('selectService', 'Select a service')}
-                id="service"
-                invalidText="Required"
-                value={queueRoomService}
-                onChange={(event) => setQueueRoomService(event.target.value)}>
-                {!queueRoomService ? <SelectItem text={t('selectService', 'Select a service')} value="" /> : null}
-                {queues?.length > 0 &&
-                  queues.map((service) => (
-                    <SelectItem key={service.uuid} text={service.display} value={service.uuid}>
-                      {service.display}
-                    </SelectItem>
-                  ))}
-              </Select>
-            </section>
-            {isMissingQueueRoomService && (
-              <section>
-                <InlineNotification
-                  style={{ margin: '0', minWidth: '100%' }}
-                  kind="error"
-                  lowContrast={true}
-                  title={t('missingQueueRoomService', 'Missing queue room service')}
-                  subtitle={t('addQueueRoomService', 'Please add a queue room service')}
-                />
-              </section>
-            )}
+            <Controller
+              name="queueRoomService"
+              control={control}
+              render={({ field: { onChange, value, ...field } }) => (
+                <Select
+                  {...field}
+                  value={value || ''}
+                  onChange={(e) => {
+                    onChange(e.target.value);
+                  }}
+                  labelText={t('queueRoomService', 'Queue Room service')}
```

Comment:
Use consistent casing:

```suggestion
                  labelText={t('queueRoomService', 'Queue room service')}
```